packages/web-components/src/components/ic-menu/ic-menu.tsx

```diff
@@ -600,24 +600,11 @@ export class Menu {
         this.preventIncorrectTabOrder = true;
         break;
       case "Backspace":
-        if (this.isSearchBar) {
-          (this.parentEl as HTMLIcSearchBarElement).setFocus();
-          if (this.searchMode === "navigation") this.setHighlightedOption(0);
-        } else if (this.isSearchableSelect) {
-          (this.parentEl as HTMLIcSelectElement).setFocus();
-        }
-        this.focusFromSearchKeypress = true;
+        this.focusOnSearchOrSelectInput(menuOptions, highlightedOptionIndex);
         break;
       default:
-        if (event.key !== "Tab") {
-          if (this.isSearchBar) {
-            (this.parentEl as HTMLIcSearchBarElement).setFocus();
-            if (this.searchMode === "navigation") this.setHighlightedOption(0);
-          } else if (this.isSearchableSelect) {
-            (this.parentEl as HTMLIcSelectElement).setFocus();
-          }
-          this.focusFromSearchKeypress = true;
-        }
+        if (event.key === "Tab") return;
```

Comment:
shouldn't this be covered by the "Tab" case on line 596?

Reply:
Yeah. That's right. The code has been updated. Since the default and backspace behaviour is the same, `case Backspace` has been removed as this should be captured by `default`